internal/controller/nnf_storage_controller.go

```diff
@@ -915,6 +938,96 @@ func (r *NnfStorageReconciler) getFsName(ctx context.Context, nnfStorage *nnfv1a
 
 }
 
+func (r *NnfStorageReconciler) runSharedMGTCommands(ctx context.Context, nnfStorage *nnfv1alpha6.NnfStorage) (*ctrl.Result, error) {
+	nnfStorageProfile, err := getPinnedStorageProfileFromLabel(ctx, r.Client, nnfStorage)
+	if err != nil {
+		return nil, dwsv1alpha3.NewResourceError("could not find pinned storage profile").WithError(err).WithFatal()
+	}
+
+	// Some tests don't fake out the nnfStorage completely
+	if len(nnfStorage.Spec.AllocationSets) == 0 {
+		return nil, nil
+	}
+
+	// If this NnfStorage is for a standalone MGT, then we don't need to run the shared MGT commands
+	if len(nnfStorage.Spec.AllocationSets) == 1 && nnfStorage.Spec.AllocationSets[0].Name == "mgt" {
```

Comment:
Would it be useful to log this information?

Reply:
Not really. This will happen whenever you're creating a new MGT for the pool. It's expected, and the MGT commands are a no-op.